pkg/chartutil/create.go

```diff
@@ -59,6 +61,18 @@ service:
   type: ClusterIP
   externalPort: 80
   internalPort: 80
+ingress:
+  enabled: false
+  # Used to create Ingress record (should used with service.type: ClusterIP).
+  # hostname: chart-example.local
```

Comment:
A hosts array declaration also seems to be more common compared to hostname:

https://github.com/kubernetes/charts/search?l=YAML&utf8=%E2%9C%93&q=hosts&type=

https://github.com/kubernetes/charts/search?l=YAML&utf8=%E2%9C%93&q=ingress.hostname&type=